src/main/python/ttconv/tt.py

```diff
@@ -36,11 +36,20 @@
 import ttconv.imsc.writer as imsc_writer
 import ttconv.srt.writer as srt_writer
 import ttconv.scc.reader as scc_reader
+from ttconv.config import GeneralConfiguration
+from ttconv.imsc.config import ImscWriterConfiguration
+from ttconv.isd import IsdConfiguration
 
 LOGGER = logging.getLogger(__name__)
 
 READING = True
 
+CONFIGURATIONS = [
+  ("general", GeneralConfiguration),
```

Comment:
It would be good for `GeneralConfiguration` to define its name, i.e. `"general"`

Reply:
See a7bc454